src/packages/core/validation/controllers/bind-server-validation-to-form-control.controller.ts

```diff
@@ -41,7 +41,7 @@ export class UmbBindServerValidationToFormControl extends UmbControllerBase {
 	}
 
 	constructor(host: UmbControllerHost, formControl: UmbFormControlMixinInterface<unknown>, dataPath: string) {
-		super(host, ctrlSymbol);
+		super(host);
```

Comment:
Maybe use this for the controllerAlias: `'umbFormControlValidation_'+simpleHashCode(dataPath)` — What do you think?

This is used to avoid the same one begin initialized twice. :-)

Reply:
I was thinking about that. My conclusion was that it would not work if multiple "things" bind to the same path - but when thinking closer - it would be a crazy thing to do :)

Do we have a "simpleHashCode" function or should we invent one?

Reply:
Found the simpleHashCode-function, it worked great. PR updated.